src/parser/monk/windwalker/modules/spells/SpinningCraneKick.tsx

```diff
@@ -1,24 +1,36 @@
 import React from 'react';
 
 import SPELLS from 'common/SPELLS';
-import Analyzer, { SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Analyzer, { Options, SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
 import AbilityTracker from 'parser/shared/modules/AbilityTracker';
 import StatTracker from 'parser/shared/modules/StatTracker';
 import Statistic from 'interface/statistics/Statistic';
 import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText/index';
 import { STATISTIC_ORDER } from 'interface/others/StatisticBox';
-import Events from 'parser/core/Events';
+import Events, { ApplyDebuffEvent, CastEvent, RefreshDebuffEvent } from 'parser/core/Events';
 
-const isEqual = (a, b) => a.id === b.id && a.instance === b.instance
+interface MarkOfTheCrane {
+  target: MarkOfTheCraneTarget,
```

Comment:
style (also applies to the interface in combostrikes): semicolons and newlines (instance isnt on a newline in the interface below this)